torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16915,6 +16938,20 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
     #
     # Elementwise Unary nn.functional OpInfos
     #
+    PythonRefInfo(
+        "_refs.nn.functional.alpha_dropout",
+        torch_opinfo_name="nn.functional.alpha_dropout",
+        supports_nvfuser=False,
+        decorators=(
+            DecorateInfo(unittest.skip("Skipped!"), 'TestCommon', 'test_python_ref'),
+            # AssertionError: Tensor-likes are not close!
```

Comment:
I think you should update that dropout output is not comparable similar to `_refs.nn.functional.dropout` skips

Reply:
Updated. Thanks, @kshitij12345!